eng/linux/package.sh

```diff
@@ -3,10 +3,29 @@
 set -eu
 . "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
 
+arch="${OTD_BUILD_ARCH:=}"
+if [ -z "$arch" ]
+then
+  echo "detecting architecture using uname..."
+  uname_m="$(uname -m)"
+  case "$uname_m" in
+    "aarch64")
+      arch="arm64";;
+    "amd64")
```

Comment:
It's `x86_64` on my system. A [stackoverflow answer](https://stackoverflow.com/a/45125525) seems to suggest that `amd64` does not exist.
```suggestion
    "x86_64")
```